### ISSUE TITLE
BirdNET Analyzer GUI - Value Error

### ISSUE THREAD

**Issue by WarblerWayne** [kahst/BirdNET-Analyzer#185]
I've been using the BirdNET Analyzer GUI to build custom classifiers to help me listen to ARD sound recordings. I've recently run into this error when trying to build train the data:
![image](https://github.com/kahst/BirdNET-Analyzer/assets/146965046/66a9f7fe-9f61-4d66-bac8-ff641aa9790c)
I've been able to successfully run the training in the past, but I'm unsure what I need to fix or change to get past this error

In my Birdnet_Custom_Classifier there are:
- 42 bird species (18 of which have the 100 recommended files for training)
- 1 Background folder for random sounds and noise (also has 100 files)
- All sound files are no longer than 3 seconds and are .WAV files
- When running the training I have stuck to the base epoch, batch size, learning rate, and hidden unit settings that the GUI provides me

Any help would be appreciated! Let me know if you need any more information.

**Comment by abfleishman** [comment]
I have no idea, but it seems like you have a file that is too short.  Are you sure all your files are 3 seconds?

**Comment by abfleishman** [comment]
The error indicates input of 0 length suggesting for one of the files.  I would make sure none are corrupt. and once you do that you can ignore my advice as I have never tried to run this process.

**Comment by Josef-Haupt** [comment]
This error can happen for several reasons. It is likely that the file is somehow corrupt and librosa can't handle it.

**Comment by max-mauermann** [comment]
I also encountered this error and like josef said it was due to corrupted files.
I will mark this as enhancement, because the training-script should not stop when trying to process these files and just skip them and give out a warning so the files can be inspected manually.

**Comment by max-mauermann** [comment]
> I also got that error message. I am wondering if there is a minimum file size? Will files <= 0.2s be padded? I am dealing with several files that are < 0.2s long, especially contact calls ...
> 
> Thank you for your great work!

From what i can tell, smaller files or chunks should be padded. I also tried it with samples that i cut down to <0.2s and they could be loaded for training and also for analyzing.

Could you provide a sample, that causes the error? that might help us find a solution
